pkg/provider/bao/configmap.go

```diff
@@ -36,12 +36,12 @@ func (m *mutator) MutateConfigMap(ctx context.Context, mutateRequest provider.Co
 	}
 	defer m.client.Close()
 
-	config := baoinjector.Config{
-		TransitKeyID:     m.config.TransitKeyID,
-		TransitPath:      m.config.TransitPath,
-		TransitBatchSize: m.config.TransitBatchSize,
-	}
-	injector := baoinjector.NewSecretInjector(config, m.client, nil, m.logger)
+	injector := baoinjector.NewSecretInjector(
+		baoinjector.Config{
+			TransitKeyID:     m.config.TransitKeyID,
+			TransitPath:      m.config.TransitPath,
+			TransitBatchSize: m.config.TransitBatchSize,
+		}, m.client, nil, m.logger)
```

Comment:
unrelated nit: could be useful to indicate type/name when passing literals
```suggestion
		}, m.client, nil /* type */, m.logger)
```